src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,19 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService isUser = new UserService();
+        User user = isUser.findByEmail(email);
+        boolean res;
+
+        if (email.equals(user.getEmail()) && password.equals(user.getPassword())) {
+            res = true;
+        } else {
+            res = false;
+        }
+
+        return res;
+
```

Comment:
remove redundant empty lines and variables
add check to see if we found a user or not
why do you need `email.equals(user.getEmail())` check